<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="Paola Pierola tests " timestamp="2017-10-30T19:06:54" hostname="localhost" time="28.105" errors="0" tests="1" skipped="0" failures="0">
  <testcase classname="Paola Pierola tests " name="should show that I can launch landing page " time="28.103">
  </testcase>
 </testsuite>
 <testsuite name="Paola Pierola  Location tests " timestamp="2017-10-30T19:07:22" hostname="localhost" time="0.033" errors="0" tests="1" skipped="0" failures="1">
  <testcase classname="Paola Pierola  Location tests " name="should show that I can launch landing page " time="0.033">
   <failure type="exception" message="Failed: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=61.0.3163.100)
  (Driver info: chromedriver=2.33.506120 (e3e53437346286c0bc2d2dc9aa4915ba81d9023f),platform=Windows NT 10.0.10586 x86_64)"><![CDATA[NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=61.0.3163.100)
  (Driver info: chromedriver=2.33.506120 (e3e53437346286c0bc2d2dc9aa4915ba81d9023f),platform=Windows NT 10.0.10586 x86_64)
    at WebDriverError (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:27:5)
    at NoSuchWindowError (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:216:5)
    at Object.checkLegacyResponse (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:109:7)
From: Task: WebDriver.navigate().to(http://www.collegeview.com/collegesearch/index.jsp?crit=0)
    at thenableWebDriverProxy.schedule (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
    at Navigation.to (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1133:25)
    at thenableWebDriverProxy.get (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:988:28)
    at ProtractorBrowser.get (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\built\browser.js:655:32)
    at LocationPage.location (G:\polet\page-objects\location.po.js:22:17)
    at UserContext.<anonymous> (G:\polet\e2e\location\location.js:20:20)
    at C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
From: Task: Run it("should show that I can launch landing page ") in control flow
    at UserContext.<anonymous> (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (G:\polet\e2e\location\location.js:13:9)
    at addSpecsToSuite (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\HP\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (G:\polet\e2e\location\location.js:8:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>